pyproject.toml

```diff
@@ -1,10 +1,97 @@
+readme = "README.rst"
+repository = "https://github.com/ansys/pyfluent"
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+]
+packages = [
+    { include = "ansys", from = "src" },
+    { include = "tests", to = "ansys/fluent" },
+    { include = "README.rst", to = "ansys/fluent/core/docs" },
+    { include = "datamodel_222", from = "src/ansys/fluent/core", to = "ansys/fluent/core" },
+    { include = "datamodel_231", from = "src/ansys/fluent/core", to = "ansys/fluent/core" },
+    { include = "datamodel_232", from = "src/ansys/fluent/core", to = "ansys/fluent/core" },
+    { include = "datamodel_241", from = "src/ansys/fluent/core", to = "ansys/fluent/core" },
+    { include = "datamodel_242", from = "src/ansys/fluent/core", to = "ansys/fluent/core" },
+    { include = "tui_222", from = "src/ansys/fluent/core/meshing", to = "ansys/fluent/core/meshing" },
+    { include = "tui_231", from = "src/ansys/fluent/core/meshing", to = "ansys/fluent/core/meshing" },
+    { include = "tui_232", from = "src/ansys/fluent/core/meshing", to = "ansys/fluent/core/meshing" },
+    { include = "tui_241", from = "src/ansys/fluent/core/meshing", to = "ansys/fluent/core/meshing" },
+    { include = "tui_242", from = "src/ansys/fluent/core/meshing", to = "ansys/fluent/core/meshing" },
+    { include = "settings_222", from = "src/ansys/fluent/core/solver", to = "ansys/fluent/core/solver" },
+    { include = "settings_231", from = "src/ansys/fluent/core/solver", to = "ansys/fluent/core/solver" },
+    { include = "settings_232", from = "src/ansys/fluent/core/solver", to = "ansys/fluent/core/solver" },
+    { include = "settings_241", from = "src/ansys/fluent/core/solver", to = "ansys/fluent/core/solver" },
+    { include = "settings_242", from = "src/ansys/fluent/core/solver", to = "ansys/fluent/core/solver" },
 ]
 
+[tool.poetry.dependencies]
+python = ">=3.9,<4.0"
+importlib-metadata = {version = "^7.0.1", python = "<3.9"}
+ansys-api-fluent = "^0.3.22"
+ansys-platform-instancemanagement = "^1.0"
+ansys-units = "^0.3.1"
+beartype = "^0.17"
+docker = "^6.1.3"
+grpcio = "^1.30.0"
+grpcio-health-checking = "^1.30.0"
+lxml = "^4.9.2"
+numpy= "^1.21.5"
+pandas = "^1.1.5"
+platformdirs = "^3.5.1"
+psutil = "^5.9.5"
+pyyaml = "^6.0"
+requests = "^2.31.0"
+
+[tool.poetry.group.docs]
+optional = true
+
+[tool.poetry.group.docs.dependencies]
+Sphinx = "^7.2.6"
+jupyter_sphinx = "^0.5.3"
+numpydoc = "^1.6.0"
+matplotlib = "^3.8.2"
+ansys-sphinx-theme = "^0.13.2"
+pypandoc = "^1.12"
+pytest-sphinx = "^0.5.0"
+sphinx-autobuild = "^2021.3.14"
+sphinx-autodoc-typehints = "^1.25.3"
+sphinx-copybutton = "^0.5.2"
+sphinx-gallery = "^0.15.0"
+sphinx-notfound-page = "^1.0.0"
+sphinxcontrib-websupport = "^1.2.7"
+sphinxemoji = "^0.3.1"
+sphinx-toggleprompt = "^0.5.2"
+autodocsumm = "^0.2.12"
```

Comment:
Our doc dependencies have version matching clause (`==`) in the requirements file and those are updated by dependabot through a PR. In past, we have seen issues when a later version is automatically picked up during PR runs. I think it is safer to retain the version matchig clause for doc dependecies (may same for tests dependencies).

Reply:
Done. Thank you.